index.js

```diff
@@ -22,10 +22,19 @@ var waitForSingleFileChange = (file, interval) =>
     checkFile();
   });
 
-module.exports = (files, interval) => {
-  if (typeof files === "string") {
-    files = [files];
+module.exports = (args) => {
```

Comment:
In that case it seems that the binary itself is somewhat ... wrong. In its current state, it's impossible to add a custom interval via CLI, since only an array is forwarded as the first and only argument.
To achieve no changes to the API, but still be able to add a custom interval value, it will be required to put the logic that splits the input to `files` and `interval` into the binary.
Would that be ok?